pal/src/map/virtual.cpp

```diff
@@ -57,6 +57,7 @@ typedef struct FREE_BLOCK {
 #endif  // MMAP_IGNORES_HINT
 
 // The first node in our list of allocated blocks.
```

Comment:
Nit: Move this comment one line below- and consider adding a new comment to describe what your new variable is